docs/documentation.md

```diff
@@ -1188,17 +1188,19 @@ If your code base is relying on an other framework, feel free to use it to creat
 
 </div>
 
+#### Example with vanillaJS
 
+To extend the capabilities of instantsearch.js, you don't need any to use any dependencies. For example, you could use only plain javascript. Have a look at our simple js example [<i class='fa fa-github'></i> instantsearch/instantsearch-JavaScript-widget](https://github.com/instantsearch/instantsearch-JavaScript-widget) to get started.
 
 #### Example with jQuery
 {:.no-toc}
 
-You may want to use an existing jQuery library to power your custom widget: that's definitely doable. Take a look at this jQuery-based custom widgets we've done: [<i class="fa fa-github"></i> algolia/instantsearch-ion.rangeSlider](https://github.com/algolia/instantsearch-ion.rangeSlider).
+You may want to use an existing jQuery library to power your custom widget: that's definitely doable. Take a look at this jQuery-based custom widgets we've done: [<i class="fa fa-github"></i> instantsearch/instantsearch-ion.rangeSlider](https://github.com/instantsearch/instantsearch-ion.rangeSlider).

```

Comment:
Can we do the same than React? => simple example, have a look at a more complete..